dags/elsevier/metadata_parser.py

```diff
@@ -101,17 +115,20 @@ def _get_license(self, article):
         return [get_license_type_and_version_from_url(url)]
 
     def _get_local_files(self, article):
-        xml = article.find("files-info/ml/pathname").text
-        pdf = {
-            "filetype": "pdf",
-            "path": os.path.join(
-                self.file_path, article.find("files-info/web-pdf/pathname").text
-            ),
-        }
-        xml = {
-            "filetype": "xml",
-            "path": os.path.join(
+        if "A.tar" in self.file_path:
+            self.file_path = self.file_path.split("A.")[:-1][0]
+        if ".zip" in self.file_path:
+            self.file_path = self.file_path.split(".zip")[:-1][0]
+        if "raw" in self.file_path:
+            self.file_path = os.path.join(*self.file_path.split("/")[1:])
```

Comment:
why not something like `os.path.dirname(os.path.realpath())` or even better take advantage of python 3?